src/common/error/src/error.rs

```diff
@@ -14,7 +14,7 @@ pub enum DaftError {
     #[error("DaftError::ComputeError {0}")]
     ComputeError(String),
     #[error("DaftError::ArrowError {0}")]
-    ArrowError(#[from] arrow2::error::Error),
+    ArrowError(String),
```

Comment:
This can still hold `arrow2::error::Error` instead of a string

Reply:
Ah gotcha, done